lib/src/components/NxTag/__tests__/NxTag.test.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+
+import { NxSelectableTag } from '../NxTag';
+
+describe('NxSelectableTag', function() {
+  const getShallowComponent = enzymeUtils.getShallowComponent(NxSelectableTag, { children: 'test' });
+
+  it('renders NxSelectableTag with the `nx-tag--selectable` class', function() {
+    const component = getShallowComponent();
+
+    expect(component).toHaveClassName('nx-tag nx-tag--selectable');
+  });
+
```

Comment:
Missing tests for caller-specified attributes and class names.

Reply:
> Missing tests for caller-specified attributes and class names.

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/9b3e4e505841a6214165df6ec0c0d3d0459a1bbd

Reply:
> > Missing tests for caller-specified attributes and class names.
> 
> 9b3e4e5

I'm not seeing that address this comment.

Reply:
> > > Missing tests for caller-specified attributes and class names.
> > 
> > 
> > [9b3e4e5](https://github.com/sonatype/sonatype-react-shared-components/commit/9b3e4e505841a6214165df6ec0c0d3d0459a1bbd)
> 
> I'm not seeing that address this comment.

I misunderstood the request, this should address it: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/ceda147b7052306a8afeb63eb12c60d29186e7a1